packages/utils/index.js

```diff
@@ -37,3 +38,13 @@ export class FabricElement extends LitElement {
 }
 
 export const windowExists = typeof window !== 'undefined';
+
+export function fclasses(definition) {
+  const defn = {};
+  for (const [key, value] of Object.entries(definition)) {
+    for (const className of key.split(' ')) {
+      defn[className] = value;
+    }
+  }
+  return classMap(defn);
+}
```

Comment:
Just a thought. Would you consider handling falsy values in case we forgot to initiate respective web component properties that stand for `value` in here? 
Also, I have a feeling that `classes` function from line 4 is not used anywhere, but that's just a side note 😄